rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,421 @@
+
+### DCP Product Backlog
+
+The DCP ZenHub board is the Product Backlog for DCP. All DCP component repos that will assign items to a DCP Release and Milestone MUST be added to the ZenHub *DCP Backlogs* workspace. The [DCP PM Team](https://github.com/orgs/HumanCellAtlas/teams/dcp-pm-team) in the Human Cell Atlas GitHub organization MUST have write access to all the repo(s) in the *DCP workspace*.
+
+The top-level *dcp* repo MUST contain **all** significant work that is important to the DCP community and requires visibility and prioritization. This includes Roadmap objectives and product-wide technical requirements (e.g. “components should have logs...“). 
+
+If Institutions are using other project management applications such as Jira for internal requirements and reporting, then all DCP related issues in a Release MUST be *projected* into ZenHub for transparency and finer granularity for tracking progress.
+
+To avoid maintenance overhead due to duplicate or incomplete issues, the creation of new items in the *dcp* repo SHOULD be limited to the PM Team or a facilitator from the Architecture Team.
+
+DCP Product Backlog items are modeled as User Stories, Epics, and Spikes. 
+ 
+#### Owners
+
+A DCP owner MUST be assigned to each issue in the Product Backlog in the *dcp* repo to coordinate progress and drive the issue to on-time closure. This owner is responsible for ensuring that related issues are created in component repo(s) and linked to Epics within the *dcp* repo.
+
+For example, an implementation task for a specific component resides in its own repo and is linked to the relevant Epic in the *dcp* repo.
+
+#### User Stories
+
+User stories are often in the form of simple templates such as:
+
+`As a <type of user>
+I want to <action>
+so that <result>`
+
+DCP issues SHOULD include a User Story, Acceptance Criteria ("Definition of Done"), and a Deliverable to be demonstrated during the Release Retrospective to Oversight.
+
+The `<type of user>` MUST be specific and reference [HCA Personas](https://docs.google.com/presentation/d/1bsu8q9CzRXv3Y8c1p8NENi3p8KlxC4scp2tJkHO3Jeo/edit#slide=id.g3cf4e46e8f_0_38) when appropriate.
+
+*As a data wrangler working on the DCP, I want to adapt the metadata schema quickly in order to respond to developments in experimental approaches and new assays.*
+
+*Acceptance Criteria*
+- *[ ] This*
+- *[ ] That*
+- *[ ] And one more thing*
+
+*Deliverable that can be demonstrated to Oversight*
+
+...
+
+### Planning and Executing a Release
+
+The DCP community members SHOULD respond to requests related to Release planning and execution in a timely manner. If there is a failure to engage and be responsive, then a gentle escalation to the DCP PM slack channel is RECOMMENDED to ensure that progress occurs and the schedule does not slip. 
```

Comment:
I've always regarded DCP as an open source community (a'la *dcp-community* repo). There's nothing blocking an unpaid volunteer from submitting a PR or a RFC. So the definition would be _working on the platform_. I discussed this a bit in the past with @adriennes. UX has stopped using  _DCP contributor_ because it sounds like someone who could be submitting data and switched to _DCP Team Member_.  I'm open to suggestions.

Reply:
Thanks for the explanation. I can't do better.